File: miniJobs/Application/Jobs/Models/JobStep2Request.cs
namespace Application.Jobs.Models;

public class JobStep2Request
{
    public int Id { get; set; }
    public int JobTypeId { get; set; }
    public JobScheduleInfo? JobSchedule { get; set; }
    public int RequiredEmployees { get; set; }
    public int ApplicationsDuration { get; set; }
}

File: miniJobs/UI/minijobs_admin/lib/models/job/job_step2.request.dart
import 'package:json_annotation/json_annotation.dart';

import 'job_schedule_info.dart';

part 'job_step2.request.g.dart';

@JsonSerializable()
class JobStep2Request {
  int? jobTypeId;
  int? requiredEmployees;
  JobScheduleInfo? jobSchedule;
  int? applicationsDuration;

  JobStep2Request(this.jobTypeId, this.requiredEmployees, this.jobSchedule, this.applicationsDuration);
  Map<String, dynamic> toJson() => _$JobStep2RequestToJson(this);
}
